general-configuration-guide/spell-checking.md

```diff
@@ -8,7 +8,7 @@ keywords: spell checker spelling browser_spellcheck gecko_spellcheck
 
 ## Browser-based spell checking
 
-Assign the [`browser_spellcheck`]({{ site.baseurl }}/configure/spelling/#browser_spellcheck) configuration option the value of `true` to utilize the browser's native spell check functionality. Enabling the [`contextmenu`]({{ site.baseurl }}/plugins/contextmenu/) option _may_ be required depending on the right-click or context usability requirement.
+Assign the [`browser_spellcheck`]({{ site.baseurl }}/configure/spelling/#browser_spellcheck) configuration option the value of `true` to utilize the browser's native spell check functionality. Enabling the [`contextmenu`]({{ site.baseurl }}/ui-components/contextmenu/) option _may_ be required depending on the right-click or context usability requirement.
```

Comment:
This link is definitely better, but I wonder if it should it be this: `{{ site.baseurl }}/configure/editor-appearance/#contextmenu`? As it's talking about enabling the `contextmenu` option.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1495/commits/34f297c361e6e7ab7c3eb5b534e236056ed6fcf4